charts/kyverno/templates/_helpers/_deployment.tpl

```diff
@@ -1,8 +1,12 @@
 {{/* vim: set filetype=mustache: */}}
 
 {{- define "kyverno.deployment.replicas" -}}
-  {{- if eq (int (default 1 .)) 0 -}}
+  {{- if not (eq . nil) -}}
+  {{- if not (kindIs "string" .) -}}
```

Comment:
Will this always work with the following conditional? What if it's a `bool`?

Reply:
But it would probably fail with a Helm error - because of the cast to int. Or am I reading this wrong?

Reply:
Anyways, thanks for the review and merge @eddycharly! 🥇

Reply:
Thanks for opening the PR @erikgb !